tools/signtool/main.go

```diff
@@ -0,0 +1,215 @@
+// Copyright 2022 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// signtool.exe emulator
+//
+// This tool emulates the signtool.exe tool from Microsoft for signing windows
+// binaries: https://msdn.microsoft.com/en-us/library/windows/desktop/aa387764(v=vs.85).aspx
+//
+// It only uses the last argument on the command-line, which is the path to the file to sign.
+// If signing is successful, it will replace the file with the newly-signed file from the
+// notary service.
+//
+// Parameters for the notary service are passed in as environment variables.
+// NOTARY_SIGNING_KEY - The name of the key to use for signing
+// NOTARY_SIGNING_COMMENT - The comment to enter into the notary log for this signing operation
+// NOTARY_AUTH_TOKEN - The password for using the selected signing key
+// NOTARY_URL - The URL of the notary service
+
+// copied from https://github.com/mongodb-js/compass/blob/b37db421f97911f6046cfbad2b4d6f0800560d53/packages/hadron-build/signtool/signtool.go
+
+package main
```

Comment:
[note] since copy/paste not investing much here, assuming getting a notarized build is good enough

Reply:
I don't follow, any actions here?

Reply:
I've tested on windows, it looks good to me

Reply:
it's just a note for traceability in case somebody comes back to look at the PR :)